projects/fsastorefrontlib/src/cms-components/agent/agent-search-box/agent-search-box.component.ts

```diff
@@ -29,4 +36,21 @@ export class AgentSearchBoxComponent extends StoreFinderSearchComponent
       this.findAgents(this.searchBox.value);
     }
   }
+
+  resetSearchValueAndRouteParams() {
```

Comment:
Private

Reply:
Let's keep it as public/protected. If a client wants to change this reset search behaviour, he would need now to override the entire onInit method or basically any method that is calling this one.